lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -64,6 +69,7 @@ internal sealed class ContextRuntime : IDisposable
                 Optional<ContextRuntime> parentContext)
         {
             _serviceInjector = serviceInjector;
+            _injectedServices = serviceInjector.GetNamedInstance<ServicesSet, ISet<object>>();
```

Comment:
When you call serviceInjector.GetInstace<ContextRuntime>, as long as you put IInjector as a parameter in ContextRuntime constructor, you would get serviceInjector for free in side ContextRuntime constructor.